components/developer/libtool/libtool.p5m

```diff
@@ -44,65 +30,63 @@ file path=usr/include/libltdl/lt_dlloader.h
-file path=usr/share/libtool/libltdl/lt__alloc.c
-file path=usr/share/libtool/libltdl/lt__dirent.c
-file path=usr/share/libtool/libltdl/lt__strl.c
-file path=usr/share/libtool/libltdl/lt_dlloader.c
-file path=usr/share/libtool/libltdl/lt_error.c
-file path=usr/share/libtool/libltdl/ltdl.c
-file path=usr/share/libtool/libltdl/ltdl.h
-file path=usr/share/libtool/libltdl/Makefile.am
-file path=usr/share/libtool/libltdl/Makefile.in
-file path=usr/share/libtool/libltdl/Makefile.inc
-file path=usr/share/libtool/libltdl/README
-file path=usr/share/libtool/libltdl/slist.c
+file path=usr/share/libtool/COPYING.LIB
+file path=usr/share/libtool/Makefile.am
+file path=usr/share/libtool/Makefile.in
+file path=usr/share/libtool/README
+file path=usr/share/libtool/aclocal.m4
+file path=usr/share/libtool/build-aux/compile
+file path=usr/share/libtool/build-aux/config.guess
+file path=usr/share/libtool/build-aux/config.sub
+file path=usr/share/libtool/build-aux/depcomp
+file path=usr/share/libtool/build-aux/install-sh
+file path=usr/share/libtool/build-aux/ltmain.sh
+file path=usr/share/libtool/build-aux/missing
+file path=usr/share/libtool/config-h.in
+file path=usr/share/libtool/configure
+file path=usr/share/libtool/configure.ac
+file path=usr/share/libtool/libltdl/lt__alloc.h
+file path=usr/share/libtool/libltdl/lt__argz_.h
+file path=usr/share/libtool/libltdl/lt__dirent.h
+file path=usr/share/libtool/libltdl/lt__glibc.h
+file path=usr/share/libtool/libltdl/lt__private.h
+file path=usr/share/libtool/libltdl/lt__strl.h
+file path=usr/share/libtool/libltdl/lt_dlloader.h
+file path=usr/share/libtool/libltdl/lt_error.h
+file path=usr/share/libtool/libltdl/lt_system.h
+file path=usr/share/libtool/libltdl/slist.h
+file path=usr/share/libtool/loaders/dld_link.c
+file path=usr/share/libtool/loaders/dlopen.c
+file path=usr/share/libtool/loaders/dyld.c
+file path=usr/share/libtool/loaders/load_add_on.c
+file path=usr/share/libtool/loaders/loadlibrary.c
+file path=usr/share/libtool/loaders/preopen.c
+file path=usr/share/libtool/loaders/shl_load.c
+file path=usr/share/libtool/lt__alloc.c
+file path=usr/share/libtool/lt__argz.c
+file path=usr/share/libtool/lt__dirent.c
+file path=usr/share/libtool/lt__strl.c
+file path=usr/share/libtool/lt_dlloader.c
+file path=usr/share/libtool/lt_error.c
+file path=usr/share/libtool/ltdl.c
+file path=usr/share/libtool/ltdl.h
+file path=usr/share/libtool/ltdl.mk
+file path=usr/share/libtool/slist.c
 file path=usr/share/man/man1/libtool.1
 file path=usr/share/man/man1/libtoolize.1
 
 depend fmri=__TBD pkg.debug.depend.file=usr/bin/ggrep type=require
 depend fmri=__TBD pkg.debug.depend.file=usr/bin/gsed type=require
 depend fmri=__TBD pkg.debug.depend.file=usr/bin/ld type=require
 depend fmri=__TBD pkg.debug.depend.file=usr/bin/nm type=require
-depend fmri=__TBD pkg.debug.depend.file=usr/ucb/echo type=require
-
-license libtool.license license="GPLv2, FDLv1.3"
+depend fmri=__TBD pkg.debug.depend.file=usr/bin/echo type=require
```

Comment:
This one is exactly not needed (seems as well as usr/ucb/echo). Now we see in libtoolize:
: ${ECHO="printf %s\n"}